samples/bluetooth/broadcast_audio_sink/prj.conf

```diff
@@ -15,4 +15,8 @@ CONFIG_BT_BUF_ACL_TX_SIZE=251
 
 CONFIG_BT_DEVICE_NAME="Broadcast Audio Sink"
 
-CONFIG_BT_TINYCRYPT_ECC=y
+# LC3 Needs these
```

Comment:
I know we should not add these, as they are board specific to whatever board we have that will be using LC3, however I need some input from reviewers on how to solve this properly.

There are two issues:

1. I have added these here since the BSIM `zephyr/tests/bsim/bluetooth/audio_samples/broadcast_audio_sink/tests_scripts/broadcast_audio.sh` will by default compile to `nrf53_bsim`, which today will not pick up the board specific conf file. 

If I do overwrite the current overlay file (which I dont want, I want to append it), it will also:

2. pick the USB component (as it should), however it will then not be able to find the USB device in the device tree. I'm not really sure how to overcome this, as there are currently no examples on how to add a dtsi file when compiling for BSIM.

And and all help would be appreciated!

@aescolar , @Thalley , @MariuszSkamra , @larsgk